.buildkite/publish_to_pypi.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# !!! WARNING DO NOT add -x to avoid leaking vault passwords
+set -euo pipefail
+
+source .buildkite/shared.sh
+
+init_python
+cd "$PACKAGE_PATH"
+python -m pip install --upgrade build twine
+python -m build
+
+export TWINE_USERNAME="__token__"
+
+# upload to test or real PyPI based on TEST_PYPI=1 env var or arg
+if [[ "${1:-}" == "TEST_PYPI=1" ]] || [[ "${TEST_PYPI:-}" =~ ^(1|TRUE|true)$ ]]; then
+  TWINE_PASSWORD=$(vault read -field publishing-api-key secret/ent-search-team/test-pypi-ent-search-dev)
```

Comment:
I'm not confident that buildkite can read secrets from `secret/ent-search-team/`. I think they need to be in `secret/ci` or similar. Is this something you've worked with the dev-prod team on?

Reply:
Oh I didn't know about that thank you! - maybe that's why [these instructions](https://github.com/elastic/python-elastic-agent-client/blob/main/PUBLISHING.md#getting-credentials) only had manual steps. You're probably right, our other secrets reference `secret/ci/elastic-connectors/...`. What's the name of the owning team / channel? I'll follow up with them.